tests/CMakeLists.txt

```diff
@@ -76,55 +75,35 @@ foreach(g_value IN ITEMS 224 112)
   )
 endforeach()
 
-# The tests write to the input file, therefore we copy it:
-add_test(
-  NAME copy_check_variants_input
-  COMMAND
-    ${CMAKE_COMMAND} -E copy_if_different
-    ${RRTMGP_DATA}/examples/rfmip-clear-sky/inputs/multiple_input4MIPs_radiation_RFMIP_UColorado-RFMIP-1-2_none.nc
-    ${CMAKE_CURRENT_BINARY_DIR}/
-)
-set_tests_properties(
-  copy_check_variants_input
-  PROPERTIES FIXTURES_REQUIRED
-             fetch_rrtmgp_data
-             FIXTURES_SETUP
-             copy_check_variants_input
-)
-
 add_test(
   NAME check_variants_lw
   COMMAND
     check_variants
-    multiple_input4MIPs_radiation_RFMIP_UColorado-RFMIP-1-2_none.nc
-    ${RRTMGP_DATA}/rrtmgp-gas-lw-g256.nc ${RRTMGP_DATA}/rrtmgp-gas-lw-g128.nc
+    ${RRTMGP_DATA}/examples/rfmip-clear-sky/inputs/multiple_input4MIPs_radiation_RFMIP_UColorado-RFMIP-1-2_none.nc
+    lw_flux_variants.nc ${RRTMGP_DATA}/rrtmgp-gas-lw-g256.nc
+    ${RRTMGP_DATA}/rrtmgp-gas-lw-g128.nc
 )
 set_tests_properties(
-  check_variants_lw
-  PROPERTIES FIXTURES_REQUIRED "fetch_rrtmgp_data;copy_check_variants_input"
+  check_variants_lw PROPERTIES FIXTURES_REQUIRED fetch_rrtmgp_data
 )
 
 add_test(
   NAME check_variants_sw
   COMMAND
     check_variants
-    multiple_input4MIPs_radiation_RFMIP_UColorado-RFMIP-1-2_none.nc
-    ${RRTMGP_DATA}/rrtmgp-gas-sw-g224.nc ${RRTMGP_DATA}/rrtmgp-gas-sw-g112.nc
+    ${RRTMGP_DATA}/examples/rfmip-clear-sky/inputs/multiple_input4MIPs_radiation_RFMIP_UColorado-RFMIP-1-2_none.nc
+    sw_flux_variants.nc ${RRTMGP_DATA}/rrtmgp-gas-sw-g224.nc
+    ${RRTMGP_DATA}/rrtmgp-gas-sw-g112.nc
 )
 set_tests_properties(
-  check_variants_sw
-  PROPERTIES FIXTURES_REQUIRED
-             "fetch_rrtmgp_data;copy_check_variants_input"
-             # The test writes to the same file as check_variants_lw, therefore
-             # we cannot run them in parallel:
-             DEPENDS
-             check_variants_lw
+  check_variants_sw PROPERTIES FIXTURES_REQUIRED fetch_rrtmgp_data
 )
 
 add_custom_target(
   validation-plots
   COMMAND
     ${Python3_EXECUTABLE} ${CMAKE_CURRENT_SOURCE_DIR}/validation-plots.py
-    --input_file multiple_input4MIPs_radiation_RFMIP_UColorado-RFMIP-1-2_none.nc
+    --state_file
+    ${RRTMGP_DATA}/examples/rfmip-clear-sky/inputs/multiple_input4MIPs_radiation_RFMIP_UColorado-RFMIP-1-2_none.nc
```

Comment:
I would specify the `--lw_vars_file` and `--sw_vars_file` arguments explicitly here. It will be easier to understand that the target requires outputs from the tests.

`TODO:` enforce running the test programs when the `validation-plots` target is requested.

Reply:
Good idea, see 9b8d000